From: Thorsten Kukuk <kukuk@thkukuk.de>
Date: Mon, 4 Apr 2016 13:48:04 +0000 (-0400)
Subject: Compile des_crypt.c and des_impl.c
X-Git-Tag: libtirpc-1-0-2-rc3~1
X-Git-Url: http://git.linux-nfs.org/?p=steved%2Flibtirpc.git;a=commitdiff_plain;h=7f6bb9a3467a57caf43425d213a06aeb7870086b

Compile des_crypt.c and des_impl.c

Add des_impl.c to become independent of deprecated functions of glibc

Fixes: f17b44048003 ('Revert commit c0547c56dafb')
Signed-off-by: Thorsten Kukuk <kukuk@thkukuk.de>
Signed-off-by: Steve Dickson <steved@redhat.com>
---

diff --git a/src/Makefile.am b/src/Makefile.am
index e4ed8aa..fba2aa4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -24,7 +24,7 @@ libtirpc_la_SOURCES = auth_none.c auth_unix.c authunix_prot.c bindresvport.c cln
         rpcb_st_xdr.c svc.c svc_auth.c svc_dg.c svc_auth_unix.c svc_auth_none.c \
 	svc_auth_des.c \
         svc_generic.c svc_raw.c svc_run.c svc_simple.c svc_vc.c getpeereid.c \
-        auth_time.c auth_des.c authdes_prot.c debug.c
+        auth_time.c auth_des.c authdes_prot.c debug.c des_crypt.c des_impl.c
 
 ## XDR
 libtirpc_la_SOURCES += xdr.c xdr_rec.c xdr_array.c xdr_float.c xdr_mem.c xdr_reference.c xdr_stdio.c xdr_sizeof.c

